solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -129,6 +126,12 @@ protected Http2SolrClient(String serverBaseUrl, Builder builder) {
     super(serverBaseUrl, builder);
 
     if (builder.httpClient != null) {
+      if (builder.followRedirects != null
```

Comment:
NOCOMMIT; it remains to be seen if we're going to retain HttpClient reuse but reusing it means we can't be customizing timeouts.